packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -531,7 +531,7 @@ const processTemplate = (relFile, inputDir, outputDir, context) => {
  * @param {*} answers
  * @param {*} param2
  */
-const runGenerator = (context, {outputDir, verbose}) => {
+const runGenerator = (context, {outputDir, templateVersion, verbose}) => {
```

Comment:
```suggestion
const runGenerator = (context, {outputDir, templateVersion = 'latest', verbose}) => {
```

Reply:
I don't really want to set that argument default in 2 places. We are using the templateVersion in main so it needs to be set there, which I'm doing. 

Normally I would agree that it should be on the function definition, but this is a private lib that we control where rungenerator is called.. and it's only being called in the main(). 

Open to conversation tho

Reply:
If we have the default value here in https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1229/files#r1209491242, I think it's better.